### ISSUE TITLE
Paradise resprite

### ISSUE THREAD

**Issue by Piras314** [Goob-Station/Goob-Station#285]
Convert a ton/all sprites to paradise.

**Comment by jellygato** [comment]
Do we need to worry about permissions from Paradise to use their sprites?

**Comment by Piras314** [comment]
No, a lot of our sprites are taken from them already and their sprites are under the same or similar license as our sprites if I remember correctly. Obviously I still need to credit them like we do for every artwork in the game.

**Comment by Piras314** [comment]
we basically did it
